docs/admin_console/event_integrations/relationships.dsl

```diff
@@ -0,0 +1,59 @@
+# Top Level event publishing
+server.api -> server.azure_service_bus.event_topic "Sends events to"
+server.events -> server.azure_service_bus.event_topic "Sends events to"
+server.api -> server.rabbit_mq.event_exchange "Sends events to"
+server.events -> server.rabbit_mq.event_exchange "Sends events to"
+
+# Azure Service Bus topics, subscriptions, and routing
+server.azure_service_bus.event_topic -> server.azure_service_bus.eventsWriteSub  "Fan-out Subscription"
+server.azure_service_bus.event_topic -> server.azure_service_bus.eventsHecSub "Fan-out Subscription"
+server.azure_service_bus.event_topic -> server.azure_service_bus.eventsWebhookSub  "Fan-out Subscription"
+server.azure_service_bus.event_topic -> server.azure_service_bus.eventsSlackSub "Fan-out Subscription"
+server.azure_service_bus.integration_topic -> server.azure_service_bus.integrationSlackSub "Subscription filtered via Slack key"
+server.azure_service_bus.integration_topic -> server.azure_service_bus.integrationWebhookSub "Subscription filtered via Webhook key"
+server.azure_service_bus.integration_topic -> server.azure_service_bus.integrationHecSub "Subscription filtered via HEC key"
+
+# EventsProcessor handling of topics/subscriptions
+server.azure_service_bus.eventsWriteSub -> server.events_processor.event_repository_handler "EventListenerService"
+server.azure_service_bus.eventsHecSub -> server.events_processor.event_integration_handler "EventListenerService"
+server.azure_service_bus.eventsSlackSub -> server.events_processor.event_integration_handler "EventListenerService"
+server.azure_service_bus.eventsWebhookSub -> server.events_processor.event_integration_handler "EventListenerService"
```

Comment:
I don't like these labels. I think you're saying the relationship is through an events service dependency, but I think relationships are best represented by verbs, not nouns. It feels like an element `EventsListenerService` exists that proxies requests. It may be that this element can be ignored, but you're still doing something like "writes events proxied through EventsListenerService" (note referencing this proxy may not even be necessary)